src/components/ClearCompletedTodosButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Todo } from '../types/Todo';
+
+type ClearCompletedTodosButtonProps = {
+  completedTodo: Todo[];
+  onDeleteTodo: (todo: Todo) => void;
+};
+
+export const ClearCompletedTodosButton = ({
+  completedTodo,
+  onDeleteTodo,
+}: ClearCompletedTodosButtonProps) => {
+  const availableCompletedTodo = completedTodo.length > 0;
```

Comment:
```suggestion
  const hasCompletedTodo = completedTodo.length > 0;
```
Name a bit confusing as you store boolean value, not todos